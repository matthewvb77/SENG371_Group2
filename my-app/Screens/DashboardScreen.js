import React from "react";
import { View, Text, StyleSheet, Image } from "react-native";

const DashboardScreen = ({ navigation }) => {
	return (
		<View className='flex items-center mt-6'>
			<Text>Dashboard</Text>
			<View className='flex items-center mt-6 bg-rose-500 p-3 rounded-full border-2 border-rose-700 w-1/2'>
				<Text
				className='font-bold text-lg text-white text-center'
				onPress={() => navigation.navigate("Roulette")}
				>
				Roulette
				</Text>
			</View>
			<View className='flex items-center mt-6 bg-rose-500 p-3 rounded-full border-2 border-rose-700 w-1/2'>
				<Text
				className='font-bold text-lg text-white text-center'
				onPress={() => navigation.navigate("Slots")}
				>
				Slots
				</Text>
			</View>
			<View className='flex items-center mt-6 bg-rose-500 p-3 rounded-full border-2 border-rose-700 w-1/2'>
				<Text
				className='font-bold text-lg text-white text-center'
				onPress={() => navigation.navigate("Spin The Wheel")}
				>
				Spin The Wheel
				</Text>
			</View>
      </View>
	);
};

export default DashboardScreen;
