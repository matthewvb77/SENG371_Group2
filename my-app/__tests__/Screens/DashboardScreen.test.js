import React from "react";

import DashboardScreen from "../../Screens/DashboardScreen";

describe("<DashboardScreen />", () => {
	it("boilerplate empty test", () => {
		// arrange
		// act
		// assert
	});
});
