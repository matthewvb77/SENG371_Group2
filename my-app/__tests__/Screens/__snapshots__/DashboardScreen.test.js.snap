// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DashboardScreen /> renders correctly 1`] = `
<View
  style={
    [
      {
        "display": "flex",
      },
      {
        "alignItems": "center",
      },
      {
        "marginTop": 24,
      },
    ]
  }
>
  <View
    style={
      [
        {
          "display": "flex",
        },
        {
          "alignItems": "center",
        },
        {
          "marginTop": 24,
        },
        {
          "paddingBottom": 12,
          "paddingLeft": 12,
          "paddingRight": 12,
          "paddingTop": 12,
        },
        {
          "width": "100%",
        },
      ]
    }
  >
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "#f43f5e",
          "borderBottomColor": "#be123c",
          "borderBottomLeftRadius": 8,
          "borderBottomRightRadius": 8,
          "borderBottomWidth": 2,
          "borderLeftColor": "#be123c",
          "borderLeftWidth": 2,
          "borderRightColor": "#be123c",
          "borderRightWidth": 2,
          "borderTopColor": "#be123c",
          "borderTopLeftRadius": 8,
          "borderTopRightRadius": 8,
          "borderTopWidth": 2,
          "display": "flex",
          "opacity": 1,
          "paddingBottom": 12,
          "paddingLeft": 12,
          "paddingRight": 12,
          "paddingTop": 12,
          "width": "50%",
        }
      }
    >
      <Text
        style={
          [
            {
              "fontWeight": "700",
            },
            {
              "fontSize": 18,
              "lineHeight": 28,
            },
            {
              "color": "#fff",
            },
            {
              "textAlign": "center",
            },
            {
              "color": "#fff",
            },
          ]
        }
      >
        Roulette
      </Text>
    </View>
  </View>
  <View
    style={
      [
        {
          "display": "flex",
        },
        {
          "alignItems": "center",
        },
        {
          "marginTop": 24,
        },
        {
          "paddingBottom": 12,
          "paddingLeft": 12,
          "paddingRight": 12,
          "paddingTop": 12,
        },
        {
          "width": "100%",
        },
      ]
    }
  >
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "#f43f5e",
          "borderBottomColor": "#be123c",
          "borderBottomLeftRadius": 8,
          "borderBottomRightRadius": 8,
          "borderBottomWidth": 2,
          "borderLeftColor": "#be123c",
          "borderLeftWidth": 2,
          "borderRightColor": "#be123c",
          "borderRightWidth": 2,
          "borderTopColor": "#be123c",
          "borderTopLeftRadius": 8,
          "borderTopRightRadius": 8,
          "borderTopWidth": 2,
          "display": "flex",
          "opacity": 1,
          "paddingBottom": 12,
          "paddingLeft": 12,
          "paddingRight": 12,
          "paddingTop": 12,
          "width": "50%",
        }
      }
    >
      <Text
        style={
          [
            {
              "fontWeight": "700",
            },
            {
              "fontSize": 18,
              "lineHeight": 28,
            },
            {
              "color": "#fff",
            },
            {
              "textAlign": "center",
            },
            {
              "color": "#fff",
            },
          ]
        }
      >
        Slots
      </Text>
    </View>
  </View>
  <View
    style={
      [
        {
          "display": "flex",
        },
        {
          "alignItems": "center",
        },
        {
          "marginTop": 24,
        },
        {
          "paddingBottom": 12,
          "paddingLeft": 12,
          "paddingRight": 12,
          "paddingTop": 12,
        },
        {
          "width": "100%",
        },
      ]
    }
  >
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "#f43f5e",
          "borderBottomColor": "#be123c",
          "borderBottomLeftRadius": 8,
          "borderBottomRightRadius": 8,
          "borderBottomWidth": 2,
          "borderLeftColor": "#be123c",
          "borderLeftWidth": 2,
          "borderRightColor": "#be123c",
          "borderRightWidth": 2,
          "borderTopColor": "#be123c",
          "borderTopLeftRadius": 8,
          "borderTopRightRadius": 8,
          "borderTopWidth": 2,
          "display": "flex",
          "opacity": 1,
          "paddingBottom": 12,
          "paddingLeft": 12,
          "paddingRight": 12,
          "paddingTop": 12,
          "width": "50%",
        }
      }
    >
      <Text
        style={
          [
            {
              "fontWeight": "700",
            },
            {
              "fontSize": 18,
              "lineHeight": 28,
            },
            {
              "color": "#fff",
            },
            {
              "textAlign": "center",
            },
            {
              "color": "#fff",
            },
          ]
        }
      >
        Spin The Wheel
      </Text>
    </View>
  </View>
  <View
    style={
      [
        {
          "display": "flex",
        },
        {
          "alignItems": "center",
        },
        {
          "marginTop": 24,
        },
        {
          "paddingBottom": 12,
          "paddingLeft": 12,
          "paddingRight": 12,
          "paddingTop": 12,
        },
        {
          "width": "100%",
        },
      ]
    }
  >
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "#374151",
          "borderBottomLeftRadius": 9999,
          "borderBottomRightRadius": 9999,
          "borderBottomWidth": 2,
          "borderLeftWidth": 2,
          "borderRightWidth": 2,
          "borderTopLeftRadius": 9999,
          "borderTopRightRadius": 9999,
          "borderTopWidth": 2,
          "display": "flex",
          "opacity": 1,
          "paddingBottom": 12,
          "paddingLeft": 12,
          "paddingRight": 12,
          "paddingTop": 12,
          "width": "50%",
        }
      }
    >
      <Text
        style={
          [
            {
              "fontWeight": "700",
            },
            {
              "fontSize": 18,
              "lineHeight": 28,
            },
            {
              "color": "#fff",
            },
            {
              "textAlign": "center",
            },
            {
              "color": "#fff",
            },
          ]
        }
      >
        Sign Out
      </Text>
    </View>
  </View>
</View>
`;
